app/views/catalog/_osd_or_embed_default.html.erb

```diff
@@ -1,5 +1,8 @@
+<% @viewer = Viewer.find_by(exhibit: @exhibit) %>
 <% if document.uploaded_resource? %>
   <%= render partial: "openseadragon_default", locals: {document: document} %>
+<% elsif @viewer.viewer_type == "mirador" %>
+  <%= render partial: "viewers/mirador", locals: {document: document} %>
 <% else %>
   <%= render partial: "oembed_default", locals: {document: document} %>
```

Comment:
Another possible solution to this could be to implement `to_partial_path` on the `Viewer` class.

That method could be something along the lines of:

```ruby
def to_partial_path
  return 'oembed_default' if viewer_type == 'SUL Embed'
  "viewers/#{viewer_type}"
end
```

Then the view can just have

```
<% else %>
  <%= render @viewer, document: document %>
<% end %>
```

That way, we can add additional viewers w/o needing to update this partial each time.